apps/settings/locales/settings.en-US.properties

```diff
@@ -1274,6 +1274,7 @@ copypaste=Copy/Paste
 multi-screen=Multi-Screen
 verbose-app-permissions=Verbose App Permissions
 pinning-the-web=Enable Pinning the Web
+view-source=Enable View Source Gesture
```

Comment:
Whoops! Looks like I rolled that into this patch:
https://github.com/mozilla-b2g/gaia/pull/31553/files#diff-81fe86f3f20787d354c9275a837572b9R127

I'll move it over here.